lib/src/main/java/com/orange/ods/compose/text/OdsText.kt

```diff
@@ -56,7 +54,6 @@ fun OdsText(
     text: String,
     style: OdsTextStyle,
     modifier: Modifier = Modifier,
-    displaySurface: OdsDisplaySurface = OdsDisplaySurface.Default,
```

Comment:
I wonder if we should remove other `displaySurface` properties in the components.

From what I see, we mainly use display surfaces to display buttons on inverted background, and if I am correct we can achieve the same effect using the `OdsTheme` method:

```kotlin
OdsTheme(themeConfiguration = OrangeThemeConfiguration(), darkThemeEnabled = !isSystemInDarkTheme()) {
    Surface {
        // ...
    }
}
```

Reply:
I created a new issue in the board for that point: #830